src/v0/util/index.js

```diff
@@ -1022,7 +1022,7 @@ function getDestinationExternalID(message, type) {
 
   if (Array.isArray(externalIdArray)) {
     externalIdArray.forEach((extIdObj) => {
```

Comment:
is forEach right for this use case? destinationExternalId will overridden right?

Reply:
Refactored this util and user .find()

Reply:
Refactored the util and replace .forEach with .find